portfolio/src/main/webapp/index.html

```diff
@@ -10,8 +10,16 @@
     <div id="content">
       <header>
         <h1>Abraham Haros's Portfolio</h1>
+        <img src="images/me/profilePicture.jpg" alt="Picture of me.">
```

Comment:
Can you make the alt text a little more descriptive? Where are you, are you smiling, etc.? This is important for screen reader users.

Reply:
You're right, I missed that, changing it.